cmd/tailscale/cli/debug.go

```diff
@@ -136,6 +136,12 @@ func debugCmd() *ffcli.Command {
 				Exec:       runLocalCreds,
 				ShortHelp:  "Print how to access Tailscale LocalAPI",
 			},
+			{
+				Name:       "raw-localapi",
+				ShortUsage: "tailscale debug raw-localapi [<method>] <path> [<body>]",
```

Comment:
How about just calling it "localapi"? I can't imagine what a not-raw variation might look like.

```suggestion
				Name:       "localapi",
				ShortUsage: "tailscale debug localapi [<method>] <path> [<body>]",
```

Reply:
done